docs/using-jellyseerr/settings/general.md

```diff
@@ -62,6 +62,12 @@ Set the default display language for Jellyseerr. Users can override this setting
 
 These settings filter content shown on the "Discover" home page based on regional availability and original language, respectively. The Streaming Region filters the available streaming providers on the media page. Users can override these global settings by configuring these same options in their user settings.
 
+## Blacklist Content with Tags and Limit Content Blacklisted per Tag
+
+These settings blacklist any TV shows or movies that have one of the entered tags. Entries are added to the blacklist using the "Process Blacklisted Tags" job. Removing blacklisted tags will remove media from the blacklist if it was blacklisted for that tag. To clear all blacklist entries created with tag, remove all tags in the setting and run the "Process Blacklisted Tags" job.  
```

Comment:
You say that "Removing blacktags will remove media from the blacklist if it was blacklisted for that blacktag".
But after you say that you need to run the "Process Blacktags" job to remove all entries.
Does the Process Blacktags job also have to run to remove a single entry? It's not really clear.

Reply:
Updated, lmk what you think!